packages/@expo/config-plugins/src/utils/Updates.ts

```diff
@@ -37,8 +37,13 @@ export function getUpdateUrl(
   return `https://exp.host/@${user}/${config.slug}`;
 }
 
-export function getAppVersion(config: Pick<ExpoConfig, 'version'>): string {
-  return config.version ?? '1.0.0';
+export function getAppVersion(
+  config: Pick<ExpoConfig, 'version'>,
+  platform: 'android' | 'ios'
+): string {
+  const version = config.version ?? '1.0.0';
+  const prefix = platform === 'android' ? 'string:' : '';
```

Comment:
I guess the preferred way to do this would be to set a string in `strings.xml` and assign the `meta-data` an `android:resource` field. We do something similar for [facebook app ID](https://github.com/expo/expo/blob/082815dcf3e2de29c76f7628844ae50a987263a6/packages/%40expo/prebuild-config/src/plugins/unversioned/expo-facebook/withAndroidFacebook.ts#L152), [navigation bar](https://github.com/expo/expo/blob/435bbba84558bdcaa37815c8ae210fb9bf4a2383/packages/expo-navigation-bar/plugin/src/withNavigationBar.ts#L143), and [name](https://github.com/expo/expo/blob/082815dcf3e2de29c76f7628844ae50a987263a6/packages/%40expo/config-plugins/src/android/Name.ts#L51).

My understanding is that this is how you ensure the value is not coerced to any other type (not 100% sure though).

One drawback to this approach would be that parsing the value could be a bit more challenging since a process would need to trace the strings file and find the matching value.

Reply:
Thanks @EvanBacon I'll look into this. We could also make what we are doing more explicit by using a prefix like `appVersion:` instead of `string:`.

Reply:
I think the `android:name` properly conveys the purpose of the `meta-data` field. The `string:` prefix appears to simply be attempting to force the type coercion to string.